pkg/services/object/put/local.go

```diff
@@ -3,13 +3,20 @@ package putsvc
 import (
 	"fmt"
 
-	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/engine"
 	"github.com/nspcc-dev/neofs-node/pkg/services/object_manager/transformer"
 	"github.com/nspcc-dev/neofs-sdk-go/object"
+	objectSDK "github.com/nspcc-dev/neofs-sdk-go/object"
 )
 
+// ObjectStorage is an object storage interface.
+type ObjectStorage interface {
+	// Put must save passed object to the local disk
```

Comment:
IMO `local disk` is too strong requirement since `Service` doesn't care about what type of storage is used.